model/span.go

```diff
@@ -96,6 +104,26 @@ func (s *Span) GetSpanKind() (spanKind trace.SpanKind, found bool) {
 	return trace.SpanKindUnspecified, false
 }
 
+func GetSpanKindFromKey(k string) trace.SpanKind {
+	if kind, ok := toSpanKind[k]; ok {
+		return kind
+	}
+	return trace.SpanKindUnspecified
+}
+
+func GetSpanKindFromStringOfSpanKind(s string) trace.SpanKind {
```

Comment:
what does this mean?

Reply:
This will change "0" (and other integral string) to `trace.SpanKind`